__bench__/iterators.re

```diff
@@ -35,7 +35,7 @@ module Seq = {
 
   let matches = (input, re) =>
     re |> exec(input)
-       |> map(result => Array.unsafeGetUnchecked(0, result |> Js.Re.matches));
+       |> map(result => Array.unsafeGetUnchecked(0, result |> Js.Re.captures));
```

Comment:
This is misleadingly named now that it's changed.

Reply:
updated in [`eed6d34` (#1)](https://github.com/glennsl/bs-revamp/pull/1/commits/eed6d34ca793a32b3d96cb5ca1f80f6ecd92886c)

Reply:
Any guidance you could provid on getting my editor to respect the formatting in this repo would be appreciated. Thank you.

Reply:
Waat you did with the `rebase` PR seems to have worked great! I don;t expect you to revert the formatting here if it's too much work though.